src/combinatorics/simple_graph/strongly_regular.lean

```diff
@@ -0,0 +1,68 @@
+/-
+Copyright (c) 2021 Alena Gusakov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Alena Gusakov
+-/
+import combinatorics.simple_graph.basic
+import data.set.finite
+/-!
+# Strongly regular graphs
+
+## Main definitions
+
+* `common_neighbors` is the intersection of the neighbor sets of two given vertices
+
+* `G.is_SRG_of n k l m` is a structure for a `simple_graph` with conditions
```

Comment:
This will add a link to the actual declaration in the HTML docs:
```suggestion
* `G.is_SRG_of n k l m` (see `is_simple_graph.is_SRG_of`) is a structure for a `simple_graph`
  satisfying the following conditions:
```